packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,20 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Whether or not to suppress non-essential logs when building Nuxt.
+   *
+   * Defaults to true when running in CI or when a TTY is not available.
+   *
+   * @type {boolean}
+   * @default {false}
+   */
+  quiet: {
+    $resolve: async (val, get) => {
+      return val ?? (isTest || isCI || !hasTTY)
```

Comment:
Lack of TTY can happen for many fallback CLIs as well (originally, it was because of WebStorm console output). We probably shouldn't assume no-tty as silent at least for default

Reply:
What are your thoughts about defaulting to `false` then? If user sets the value, then this value will be set globally, otherwise the false will be set

Reply:
Makes sense. We could also only disable for test environments

Reply:
I am not sure about it. I would keep it disabled globally by default and just give users the option to enable it when needed (for example based on the value of environment variable)

Reply:
We do disable logs for built-in tests currently by default. `isTest` only makes sense to be only for consistent behavior.

```suggestion
      return val ?? (isTest)
```